### ISSUE TITLE
feature request: persist the filters selection when searching

### ISSUE THREAD

**Issue by yuderekyu** [lemeryfertitta/Climbdex#14]
sick repo ✨ curious if we could convert the filters selection page to a persistent filters search bar that appears above the results page? this would make it easier for a user to update their search criteria without having to navigate between the pages.

**Comment by lemeryfertitta** [comment]
I originally had this as a one page app. I found that it felt like the page started to get cluttered and hard to navigate, so I broke it apart. I've been using the app while boarding, and I personally find it pretty easy to update the search criteria by hitting the back button (either via browser or the small link on the results page), but I'm open to a change if there is a clean design. Did you have any particular design in mind? Maybe a screenshot, diagram, or similar could help.

Just looking at Google now, Climbdex seems a little trickier because there are more options, and there has to be space for the large, collapsible hold filter.
